src/fastapi_server.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3
+# coding: utf-8
+
+"""
+Manual test:
+curl localhost:5000/join/veso
+curl localhost:5000/join/ilian
+curl localhost:5000/join/lucho
+curl localhost:5000/join/marto
+
+curl localhost:5000/players
+
+curl localhost:5000/state
+
+curl -H 'user-token:<user-token>' localhost:5000/play/roll
+
+"""
+
+from fastapi import FastAPI
+from fastapi import Header, HTTPException
+from fastapi.responses import JSONResponse
+
+import dataclasses, json
+import uuid
+
+from settings import host, port
+
+from state import Board, GameState, GameMove
+from engine import GameEngine
+from typing import Dict, Any
+
+static_path = "."
+
+app = FastAPI()
+
+player_token_name: Dict[str, str] = {}
```

Comment:
👋 @mapto , how would you describe the purpose of each of the player variables? I couldn't tell just by their names.

Reply:
It seems to be all possible mappings between token and player name or number. For a start player_token_name is not really used. @luchesar, you mind if we clean up a bit?